components/Map/index.js

```diff
@@ -6,15 +6,61 @@ import "leaflet-defaulticon-compatibility/dist/leaflet-defaulticon-compatibility
 import "leaflet-defaulticon-compatibility";
 import LocationMarker from "./LocationMarker";
 import { CustomerStock } from "../AddDeposit/styles";
+import { IconContext } from "react-icons";
+import {
+  PiCellSignalFullFill,
+  PiCellSignalHighFill,
+  PiCellSignalMediumFill,
+  PiCellSignalLowFill,
+} from "react-icons/pi";
+import { renderToStaticMarkup } from "react-dom/server";
+import CustomerNavigation from "../CustomerNavigation/CustomerNavigation";
 
 const StyledMapContainer = styled(MapContainer)`
   height: 45rem;
   width: 100%;
-  margin-top: 100px;
+  margin-top: 90px;
   z-index: 0;
 `;
+const StyledLegend = styled.div`
+  position: fixed;
+  top: 5px;
+  right: 20px;
+  background-color: white;
+  padding: 10px;
+  border-radius: 5px;
+  z-index: 2000;
+  font-size: 12px;
+  &:hover {
+    box-shadow: 0 4px 8px rgba(0, 0, 0, 0.3);
+  }
+`;
 
-const goldIcon = new L.Icon({
+const StyledPopup = styled(Popup)`
+  background-color: ${(props) => props.theme.third};
+  z-index: 1005;
+`;
+const yellowMarker = new L.Icon({
```

Comment:
Only ``L`` is a bit cryptic, isn't it? Maybe ``Leaflet`` is an alternative?

Reply:
Hey, i just took it from the template, i will have a look how to change this.